app/src/main/cpp/Tray.cpp

```diff
@@ -100,10 +100,10 @@ namespace {
 
     bool Intersect(const vrb::Vector& aStartPoint, const vrb::Vector& aDirection, vrb::Vector& aResult, bool& aInside) {
       const float dotNormals = aDirection.Dot(colliderNormal);
-//      if (dotNormals > -kEpsilon) {
-//        // Not pointed at the plane
-//        return false;
-//      }
+      if (dotNormals < -kEpsilon) {
```

Comment:
I don't think this is correct. Maybe the normals for the tray are wrong? For the intersection ray to even been in the same plane the angle between them must be obtuse. Since the collision ray should be pointed into the screen and the normal for the surface should be pointing out of the screen. If the normal for the surface points into the screen too, then the poly shouldn't even be visible.